src/components/YieldPools/ElasticFarmGroup/index.tsx

```diff
@@ -34,7 +34,12 @@ import { formatDollarAmount } from 'utils/numbers'
 
 import { ClickableText, ProMMFarmTableHeader } from '../styleds'
 import Row, { Pool } from './Row'
-import { ConnectWalletButton, DepositButton, ForceWithdrawButton, HarvestAllButton, WithdrawButton } from './buttons'
+import {
+  ConnectWalletButton,
+  DepositButton, //ForceWithdrawButton,
```

Comment:
```suggestion
  DepositButton,
```

Reply:
keep it for the future, in case contract have issue, we need to enable it again